flowbased-computation/flowbased-computation-impl/src/test/java/com/farao_community/farao/flowbased_computation/impl/MinRamAdjustmentTest.java

```diff
@@ -0,0 +1,118 @@
+
+    private MinRamAdjustment minRamAdjustment;
+    private MinRamAdjustment minRamAdjustmentBis;
+
+    @Before
+    public void setUp() {
+        FileSystem fileSystem = Jimfs.newFileSystem(Configuration.unix());
+        InMemoryPlatformConfig platformConfig = new InMemoryPlatformConfig(fileSystem);
+        platformConfig.createModuleConfig("load-flow").setStringProperty("default", "MockLoadflow");
+
+        network = MinRamAdjustmentExampleGenerator.network();
+        cracFile = MinRamAdjustmentExampleGenerator.cracFile();
+        glskProviderCore = MinRamAdjustmentExampleGenerator.glskProviderCore();
+        glskProviderAll = MinRamAdjustmentExampleGenerator.glskProviderAll();
+        countries = Arrays.asList("FR", "BE", "DE", "NL");
+        computationManager = LocalComputationManager.getDefault();
+        parameters = FlowBasedComputationParameters.load(platformConfig);
+
+        frmById = cracFile.getPreContingency().getMonitoredBranches().stream()
+                .collect(Collectors.toMap(MonitoredBranch::getBranchId, monitoredBranch -> monitoredBranch.getFmax() * FRM_DEFAULT_ON_FMAX));
+
+        ramrById = cracFile.getPreContingency().getMonitoredBranches().stream()
+                .collect(Collectors.toMap(MonitoredBranch::getBranchId, monitoredBranch -> RAMR_DEFAULT));
+
+        ramrByIdBis = cracFile.getPreContingency().getMonitoredBranches().stream()
+                .collect(Collectors.toMap(MonitoredBranch::getBranchId, monitoredBranch -> RAMR_DEFAULT_BIS));
+
+        SensitivityComputationFactory sensitivityComputationFactory = MinRamAdjustmentExampleGenerator.sensitivityComputationFactory();
+        LoadFlow.Runner loadFlowRunner = Mockito.mock(LoadFlow.Runner.class);
+        Mockito.when(loadFlowRunner.run(Mockito.any(), Mockito.any(), Mockito.any(), Mockito.any())).thenReturn(new LoadFlowResultImpl(true, Collections.emptyMap(), ""));
+        LoadFlowService.init(loadFlowRunner, computationManager);
+        SensitivityComputationService.init(sensitivityComputationFactory, computationManager);
+
+        minRamAdjustment = new MinRamAdjustment(network, cracFile, glskProviderCore, glskProviderAll, frmById, ramrById, countries, computationManager, parameters);
+        minRamAdjustmentBis = new MinRamAdjustment(network, cracFile, glskProviderCore, glskProviderAll, frmById, ramrByIdBis, countries, computationManager, parameters);
+
+    }
+
+    @Test
+    public void testRun() {
+        Map<String, Double> resultAmr = minRamAdjustment.calculateAMR();
+        assertTrue(!resultAmr.isEmpty());
+        for (String branch : resultAmr.keySet()) {
+            assertNotNull(resultAmr.get(branch));
+        }
+
+        Map<String, Double> resultAmrBis = minRamAdjustmentBis.calculateAMR();
+        assertTrue(!resultAmrBis.isEmpty());
+        for (String branch : resultAmrBis.keySet()) {
+            assertNotNull(resultAmrBis.get(branch));
+        }
+    }
```

Comment:
could we assert that values are equal to some results here ?
instead of `assertNotNull`

Reply:
removed